pkg/webhook/webhook.go

```diff
@@ -254,61 +348,47 @@ func (wh *WebHook) selfDeregistration(webhookConfigName string) error {
 func mutatePods(
 	review *admissionv1beta1.AdmissionReview,
 	lister crdlisters.SparkApplicationLister,
-	sparkJobNs string) *admissionv1beta1.AdmissionResponse {
+	sparkJobNs string) (*admissionv1beta1.AdmissionResponse, error) {
 	if review.Request.Resource != podResource {
```

Comment:
Looks like we don't need this check here any more as the same check is added above.